pyscf/adc/radc.py

```diff
@@ -529,12 +554,20 @@ def kernel_gs(self):
         mem_incore = (max(nao_pair**2, nmo**4) + nmo_pair**2) * 8/1e6
         mem_now = lib.current_memory()[0]
 
-        if (self._scf._eri is None or
-            (mem_incore+mem_now >= self.max_memory and not self.incore_complete)):
+        if getattr(self, 'with_df', None) or getattr(self._scf, 'with_df', None):  
+           if getattr(self, 'with_df', None): 
+               self.with_df = self.with_df
```

Comment:
Is this setting self.with_df to True? If not, what is it setting self.with_df to?